docs/src/appendix/cheatcodes.md

```diff
@@ -6,3 +6,12 @@
 * [`stop_roll`](cheatcodes/stop_roll.md)
 * [`start_warp`](cheatcodes/start_warp.md)
 * [`stop_warp`](cheatcodes/stop_warp.md)
+
+> ℹ️ **Info**
```

Comment:
Is this added to changelog?
When do we plan to remove this warning, do we have an issue for it?